scripts/dockerfiles/Dockerfile.cleanbuild

```diff
@@ -14,12 +14,9 @@
 FROM golang:1.9
 MAINTAINER Amazon Web Services, Inc.
 
-RUN mkdir /out
-VOLUME ['/out']
-
-RUN mkdir -p /go/src/github.com/aws/
-
 COPY ./scripts/cleanbuild /scripts/cleanbuild
 WORKDIR /go/src/github.com/aws/amazon-ecs-agent
 
+# Preserve permissions.. https://github.com/docker-library/golang/blob/2f2f3b620d61f533484f24a568c2ca46e4fda91c/1.9/stretch/Dockerfile#L49
```

Comment:
That's not preserving, but setting.  Those perms are fine, but the comment is inaccurate.

Reply:
WRITEABLE EVERYWHERE